swagger.json

```diff
@@ -1562,9 +1582,57 @@
           }
         }
       },
+      "get": {
```

Comment:
We completely missed the backend to get a list of all user-defined process graphs.